fe/fe-core/src/test/java/org/apache/doris/planner/QueryPlanTest.java

```diff
@@ -407,6 +407,16 @@ public static void beforeClass() throws Exception {
 
         createView("create view test.tbl_null_column_view AS SELECT *,NULL as add_column  FROM test.test1;");
 
+        createTable("CREATE TABLE test.table_or(" +
```

Comment:
There is no need to create a new table for every test. We could reuse the existing table in the test class, e.g, `test.test1`.